src/Docker.DotNet/Models/Events.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Docker.DotNet.Models
+{
+    public class EventMessage
```

Comment:
I got it from here https://github.com/docker/docker/blob/master/api/types/events/events.go which are the only objects used by this method. Do you want submit a PR with the Go stuff changed? I don't have it here and would take a while to get it working...